Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -1193,24 +1204,35 @@ public static void HandleHousekeep(Session session, params string[] parameters)
         }
 
         // idlist
-        [CommandHandler("idlist", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 0)]
+        [CommandHandler("idlist", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 0,
+            "Shows the next ID that will be allocated from SQL.")]
         public static void HandleIDlist(Session session, params string[] parameters)
```

Comment:
No, it is just adding 1 to the current player's Id:
`ObjectGuid nextPlayerGuid = new ObjectGuid(currentPlayerGuid.Full + 1);`

It is only correct if you're on the last character the server made.  As is, it is logically incorrect with the stated purpose from the documentation.  Even if it were correct, I don't see how this is useful.